### DISCUSSION TITLE
Add custom sound effect for Dark Whisper

### DISCUSSION THREAD

**Discussion by Scooby-Goober** [Jagusti/fvtt-wfrp4e-gmtoolkit#258] [Ideas]
My players frequently miss Dark Whisper notifications because they are tabbed out or just not paying attention to the chat, and I like to keep Dark Whispers secret if possible so it can be awkward to get their attention without giving away the game. 

The current sound effect that play when a player receives a Dark Whisper I think is just the default chat sound effect, it's quiet and easy to miss. It'd be cool if we could customize it like you can in the Sweet Nothings module

Thanks!

**Comment by Jagusti** [comment]
Oh, I like that idea. No clue how I'd do it 😆 but yea, when I get back to development on this I'll look into it. Thanks!
